### DISCUSSION TITLE
MAP not executing app

### DISCUSSION THREAD

**Discussion by KnightCoder** [Project-MONAI/monai-deploy-app-sdk#455] [Q&A]
The MAP package creates the docker image but while running it does not run the app. It executes and then exits (without any error). Please guide what could've gone wrong. I also noticed that the python `import` statements to import dependent python packages are executing. Also, the app executes when we run the `monai-deploy exec app -i /input/path/` .

MONAI Deploy SDK version: 0.5.1
Referring doc: https://docs.monai.io/projects/monai-deploy-app-sdk/en/0.5.1/developing_with_sdk/packaging_app.html

**Comment by MMelQin** [comment]
@KnightCoder I think this is the same issue with the[ newer one](https://github.com/Project-MONAI/monai-deploy-app-sdk/discussions/456), which was answered.
